lib/add_product_page/add_product_page.dart

```diff
@@ -0,0 +1,430 @@
+                      ),
+                      Expanded(
+                        flex: 4,
+                        child: TextField(
+                          decoration: InputDecoration(
+                              hintText: "Introduceți culoarea",
+                              hintStyle: TextStyle(fontSize: 14),
+                              border: InputBorder.none),
+                          style: TextStyle(fontSize: 14),
+                        ),
+                      )
+                    ],
+                  ),
+                ),
+                Container(
+                  decoration: BoxDecoration(color: Colors.white),
+                  child: Row(
+                    children: [
+                      Expanded(
+                        flex: 1,
+                        child: Center(
+                          child: Text("Mărime"),
+                        ),
+                      ),
+                      Expanded(
+                        flex: 4,
+                        child: TextField(
+                          decoration: InputDecoration(
+                              hintText: "Introduceți mărimea",
+                              hintStyle: TextStyle(fontSize: 14),
+                              border: InputBorder.none),
+                          style: TextStyle(fontSize: 14),
+                        ),
+                      )
+                    ],
+                  ),
+                ),
+                Container(
+                  decoration: BoxDecoration(color: Colors.black12),
+                  child: Row(
+                    children: [
+                      Expanded(
+                        flex: 1,
+                        child: Center(
+                          child: Text("Finisaj"),
+                        ),
+                      ),
+                      Expanded(
+                        flex: 4,
+                        child: TextField(
+                          decoration: InputDecoration(
+                              hintText: "Introduceți tipul de finisaj",
+                              hintStyle: TextStyle(fontSize: 14),
+                              border: InputBorder.none),
+                          style: TextStyle(fontSize: 14),
+                        ),
+                      )
+                    ],
+                  ),
+                ),
+                Container(
+                  decoration: BoxDecoration(color: Colors.white),
+                  child: Row(
+                    children: [
+                      Expanded(
+                        flex: 1,
+                        child: Center(
+                          child: Text("Brand"),
+                        ),
+                      ),
+                      Expanded(
+                        flex: 4,
+                        child: TextField(
+                          decoration: InputDecoration(
+                              hintText: "Introduceți brand-ul",
```

Comment:
"brandul" Regula e ca daca ultima litera e exact si sunetul care se aude ultimul, atunci e fara liniuta, doar in cazul in care spre exemplu ai "site", in limba romana ultimul sunet care se aude nu e "e" si atunci se foloseste "site-ul".